src/components/mote/components/innkalling/NyttDialogMote.tsx

```diff
@@ -0,0 +1,144 @@
+        onRequestClose={() => setBehandlerModalIsOpen(false)}
+        closeButton={true}
+        contentLabel="Skal behandler være med"
+        ariaHideApp={false}
+      >
+        <ModalContentContainer>
+          <FlexRow>
+            <h3>Skal behandleren være med i dialogmøtet?</h3>
+          </FlexRow>
+
+          <FlexRow>
+            <FlexColumn>
+              <Knapp
+                onClick={() => {
+                  setBehandlerModalIsOpen(false);
+                  history.push(`/sykefravaer/${fnr}/mote`);
+                }}
+              >
+                Ja
+              </Knapp>
+            </FlexColumn>
+            <FlexColumn>
+              <Knapp
+                onClick={() => {
+                  setBehandlerModalIsOpen(false);
+                  setNyLosningModalIsOpen(true);
+                }}
+              >
+                Nei
+              </Knapp>
+            </FlexColumn>
+            <FlexColumn>
+              <Flatknapp onClick={() => setBehandlerModalIsOpen(false)}>
+                Avbryt
+              </Flatknapp>
+            </FlexColumn>
+          </FlexRow>
+        </ModalContentContainer>
+      </ModalWrapper>
+
+      <ModalWrapper
+        isOpen={nyLosningModalIsOpen}
+        onRequestClose={() => setNyLosningModalIsOpen(false)}
+        closeButton={true}
+        contentLabel="Ny løsning for innkalling til Dialogmøte"
+        ariaHideApp={false}
+      >
+        <ModalContentContainer>
+          <FlexRow>
+            <Veileder
+              center
+              tekst={
+                <ModalText>
+                  Vi utvikler en ny løsning der du kan kalle inn til dialogmøte
+                  og skrive referat direkte i Modia.
+                </ModalText>
+              }
+              posisjon="høyre"
+            >
+              <img alt="Bilde av arbeider" src={ArbeiderKvinneImage} />
+            </Veileder>
+          </FlexRow>
+
+          <FlexRow>
+            <h3>Ønsker du å prøve ut den nye løsningen?</h3>
+          </FlexRow>
+
+          <FlexRow>
+            <FlexColumn>
+              <Knapp
+                onClick={() => {
+                  setNyLosningModalIsOpen(false);
+                  history.push(`/sykefravaer/${fnr}/dialogmote`);
+                }}
+              >
+                Ja
+              </Knapp>
+            </FlexColumn>
+            <FlexColumn>
+              <Knapp
+                onClick={() => {
+                  setNyLosningModalIsOpen(false);
+                  history.push(`/sykefravaer/${fnr}/mote`);
+                }}
+              >
+                Nei
+              </Knapp>
+            </FlexColumn>
+            <FlexColumn>
+              <Flatknapp onClick={() => setNyLosningModalIsOpen(false)}>
+                Avbryt
+              </Flatknapp>
+            </FlexColumn>
+          </FlexRow>
+        </ModalContentContainer>
+      </ModalWrapper>
+    </>
+  );
+
+  return erLokal() ? dm2 : dm1;
```

Comment:
Skulle vi gjort den tilgjengelig i preprod også? (ala sjekken for visInnkallingDialogmote i AppRouter. kanskje legge sjekken/konstanten et globalt sted i så fall)

Reply:
Godt innspill, det kan jeg fikse